test/JDBC/input/collation_tests-vu-prepare.sql

```diff
@@ -0,0 +1,118 @@
+--varchar
+CREATE TABLE collation_tests_english_collation_varchar (name varchar(20) COLLATE latin1_general_ci_as);
```

Comment:
naming convention is filename_xxx, please follow for all the objects

Reply:
Updated in the latest commit.